rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/ServiceConstants.java

```diff
@@ -32,6 +32,7 @@
 
     public static final String CLASS_NOT_FOUND = "Class not found";
     public static final String FUNCTION_NOT_FOUND = "Function not found";
+    public static final String PROPERTY_NOT_FOUND = "Property not found";
```

Comment:
Is this a duplicated line?

Reply:
They look like different lines to me - one is used when a Function isn't found and one is used when a Property is not found.